### PR TITLE
Butenko

### PR DESCRIPTION
**Pull request by kioko66**

### CODE REVIEW

File: src/psu/kit/tanya/TanyaCalculator.java [hunk 1]
```diff
@@ -1,13 +1,56 @@
...
+        int whoopsCount = 0;
+        int yayCount = 0;
+
+        int[] countT = new int[52];
+        int[] countS = new int[52];
+
+        countLetters(countS, s);
+        countLetters(countT, t);
+
+        for (int i = 0; i < 26; i++) {
```
Comment (xdemonx) [inline src/psu/kit/tanya/TanyaCalculator.java]: magic const

File: src/psu/kit/tanya/TanyaCalculator.java [hunk 2]
```diff
@@ -1,13 +1,56 @@
...
 
-
     public TanyaReaction calculate(String s, String t) {
 
-        throw new UnsupportedOperationException("Метод ещё не реалиозован");
+        int whoopsCount = 0;
+        int yayCount = 0;
+
+        int[] countT = new int[52];
+        int[] countS = new int[52];
```
Comment (xdemonx) [inline src/psu/kit/tanya/TanyaCalculator.java]: magic const 52

File: src/psu/kit/tanya/TanyaCalculator.java [hunk 3]
```diff
@@ -1,13 +1,57 @@
 package psu.kit.tanya;
 
-import java.util.Scanner;
-
 public class TanyaCalculator {
 
-
+    public int magicConst = 26;
```
Comment (xdemonx) [inline src/psu/kit/tanya/TanyaCalculator.java]: no

**Comment by maskbor** [comment]
done

**Pull request closed without merging**